src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -6662,55 +6759,43 @@ void Test(int[] a)
 ");
     }
 
-    [Theory]
-    [CombinatorialData]
-    public void Subsumption_Slice_00(
-        [CombinatorialValues(
-                "[1,2,3]",
-                "[1,2,3,..[]]",
-                "[1,2,..[],3]",
-                "[1,..[],2,3]",
-                "[..[],1,2,3]",
-                "[1,..[2,3]]",
-                "[..[1,2],3]",
-                "[..[1,2,3]]",
-                "[..[..[1,2,3]]]",
-                "[..[1,2,3,..[]]]",
-                "[..[1,2,..[],3]]",
-                "[..[1,..[],2,3]]",
-                "[..[..[],1,2,3]]",
-                "[..[1,..[2,3]]]",
-                "[..[..[1,2],3]]",
-                "[1, ..[2], 3]",
-                "[1, ..[2, ..[3]]]",
-                "[1, ..[2, ..[], 3]]")]
-            string case1,
-        [CombinatorialValues(
-                "[1,2,3]",
-                "[1,2,3,..[]]",
-                "[1,2,..[],3]",
-                "[1,..[],2,3]",
-                "[..[],1,2,3]",
-                "[1,..[2,3]]",
-                "[..[1,2],3]",
-                "[..[1,2,3]]",
-                "[..[..[1,2,3]]]",
-                "[..[1,2,3,..[]]]",
-                "[..[1,2,..[],3]]",
-                "[..[1,..[],2,3]]",
-                "[..[..[],1,2,3]]",
-                "[..[1,..[2,3]]]",
-                "[..[..[1,2],3]]",
-                "[1, ..[2], 3]",
-                "[1, ..[2, ..[3]]]",
-                "[1, ..[2, ..[], 3]]")]
-            string case2)
+    [Fact]
+    public void Subsumption_Slice_00()
     {
-        var src = @"
-using System;
+        const int Count = 18;
```

Comment:
I don't understand why were using Random here. Why are we not just testing everything?

Reply:
The original test takes too long even with a pairwise setting. This takes 1-2 sec on my machine. I think that's better than skipping it entirely.

Reply:
I see. Consider documenting it as such.